extension/src/plots/model/index.ts

```diff
@@ -330,6 +339,20 @@ export class PlotsModel extends ModelWithPersistence {
     this.fetchedRevs.delete(id)
   }
 
+  private getCLIIdToLabel() {
+    const mapping: { [shortSha: string]: string } = {}
+
+    for (const rev of this.getSelectedRevisions()) {
+      mapping[this.getCLIId(rev)] = rev
+    }
+
+    return mapping
+  }
+
+  private getCLIId(label: string) {
+    return this.branchRevisions[label] || label
```

Comment:
[F] At the moment this is the only substitution that we do but we can extend this if we have to.